sdk/storage/storage-queue/recordings/browsers/queueserviceclient/recording_can_create_and_delete_a_queue.json

```diff
@@ -2,25 +2,25 @@
  "recordings": [
   {
    "method": "PUT",
-   "url": "https://fakestorageaccount.queue.core.windows.net/queue157120301266007758",
+   "url": "https://fakestorageaccount.queue.core.windows.net/queue161289997619409672",
```

Comment:
A general comment on all the recordings.. 
I just skimmed through the changes in the recordings, and it looks like they don't need to be regenerated, the older recordings would work just fine.

I recommend reverting the changes to the recordings in this PR since they are not needed.

Reply:
There were 3 such recordings. I removed them. Apart from them, I have validated all other recordings and confirmed that they are needed.

Reply:
I see, thanks for confirming.